mcs_benchmark_data/pipelines/physical_iqa/physical_iqa_benchmark_transformer.py

```diff
@@ -0,0 +1,146 @@
+import json
+import os
+from datetime import datetime
+from pathlib import Path
+from typing import Tuple, Generator
+from rdflib import URIRef
+from dataclasses_json import dataclass_json
+
+from mcs_benchmark_data._model import _Model
+from mcs_benchmark_data._transformer import _Transformer
+from mcs_benchmark_data.models.benchmark import Benchmark
+from mcs_benchmark_data.models.benchmark_bootstrap import BenchmarkBootstrap
+from mcs_benchmark_data.models.benchmark_hypothesis import BenchmarkHypothesis
+from mcs_benchmark_data.models.benchmark_concept import BenchmarkConcept
+from mcs_benchmark_data.models.benchmark_context import BenchmarkContext
+from mcs_benchmark_data.models.benchmark_dataset import BenchmarkDataset
+from mcs_benchmark_data.models.benchmark_train_dataset import BenchmarkTrainDataset
+from mcs_benchmark_data.models.benchmark_test_dataset import BenchmarkTestDataset
+from mcs_benchmark_data.models.benchmark_dev_dataset import BenchmarkDevDataset
+from mcs_benchmark_data.models.benchmark_goal import BenchmarkGoal
+from mcs_benchmark_data.models.benchmark_question_type import BenchmarkQuestionType
+from mcs_benchmark_data.models.benchmark_sample import BenchmarkSample
+
+
+class PhysicalIQaBenchmarkTransformer(_Transformer):
+
+    __URI_BASE = "benchmark:physicaliqa"
```

Comment:
Duplicating this.

How about creating an ABC _BenchmarkTransformer that has
```
@property
def _uri_base(self):
   return f"benchmark:{self._pipeline_id}"
```
or something like that.